bot.py

```diff
@@ -63,6 +65,42 @@ def get_token():
 		return "".join(f.readlines()).strip()
 
 
+@bot.command()
+async def minigame(ctx: commands.Context):
```

Comment:
If you want the minigame to appear randomly, unprompted, then you don't want to rely on `ctx`. Would recommend moving this code to a new function that is not a bot command.